frontend/src/views/Apps/AppDetail/AppDetail.js

```diff
@@ -0,0 +1,205 @@
+import React, {Component} from "react";
+import BootstrapTable from "react-bootstrap-table-next";
+import {Alert, Button, Col, Row, Badge} from "react-bootstrap";
+import InfoCard from "../../../core/components/InfoCard/InfoCard";
+import HelpLink from "../../../core/components/HelpLink";
+import {NETWORK_TABLE_COLUMNS} from "../../../constants";
+import "./AppDetail.scss";
+import ApplicationService from "../../../core/services/PocketApplicationService";
+import NetworkService from "../../../core/services/PocketNetworkService";
+
+class AppDetail extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      pocketApplication: {},
+      networkData: {},
+      chains: [],
+    };
+  }
+
+  async componentDidMount() {
+    // eslint-disable-next-line react/prop-types
+    const {address} = this.props.match.params;
+
+    const {
+      pocketApplication,
+      networkData,
+    } = await ApplicationService.getApplication(address);
+
+    const chains = await NetworkService.getNetworkChains(networkData.chains);
+
+    this.setState({pocketApplication, networkData, chains});
+  }
+
+  render() {
+    const {
+      name,
+      url,
+      contactEmail,
+      description,
+      icon,
+    } = this.state.pocketApplication;
+    const {
+      jailed,
+      max_relays,
+      staked_tokens,
+      status,
+      public_key,
+      address,
+    } = this.state.networkData;
+
+    let statusCapitalized = "";
+
+    if (status) {
+      statusCapitalized = status[0].toUpperCase() + status.slice(1);
+    }
+
+    const {chains} = this.state;
+
+    const generalInfo = [
+      {title: `${staked_tokens} POKT`, subtitle: "Stake tokens"},
+      {title: statusCapitalized, subtitle: "Stake status"},
+      {title: max_relays, subtitle: "Max Relays"},
+    ];
+
+    const contactInfo = [
+      {title: url, subtitle: "URL"},
+      {title: contactEmail, subtitle: "Email"},
+    ];
+
+    const exJson = {
+      version: "0.0.1",
+      app_address: "bd4a...",
+      client_pub_key: "9948...",
+      signature: "a383...",
+    };
```

Comment:
Please add a TODO here to delete this value on the future, we have an endpoint to retrieve this information.